diff --git a/node_modules/react-native/ReactAndroid/build.gradle.kts b/node_modules/react-native/ReactAndroid/build.gradle.kts
index 4909aed..4de7719 100644
--- a/node_modules/react-native/ReactAndroid/build.gradle.kts
+++ b/node_modules/react-native/ReactAndroid/build.gradle.kts
@@ -535,10 +535,12 @@ android {
             "-DREACT_BUILD_DIR=$buildDir",
             "-DANDROID_STL=c++_shared",
             "-DANDROID_TOOLCHAIN=clang",
+            "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON",
             // Due to https://github.com/android/ndk/issues/1693 we're losing Android
             // specific compilation flags. This can be removed once we moved to NDK 25/26
             "-DANDROID_USE_LEGACY_TOOLCHAIN_FILE=ON")
 
+
         targets(
             "jsijniprofiler",
             "reactnativeblob",
